tests/ui/invalid_frozen_pyclass_borrow.stderr

```diff
@@ -16,6 +16,19 @@ note: required by a bound in `extract_pyclass_ref_mut`
    | pub fn extract_pyclass_ref_mut<'a, 'py: 'a, T: PyClass<Frozen = False>>(
    |                                                        ^^^^^^^^^^^^^^ required by this bound in `extract_pyclass_ref_mut`
 
+error[E0271]: type mismatch resolving `<Foo as PyClass>::Frozen == False`
+ --> tests/ui/invalid_frozen_pyclass_borrow.rs:9:1
+  |
+9 | #[pymethods]
+  | ^^^^^^^^^^^^ expected `False`, found `True`
+  |
+note: required by a bound in `PyRefMut`
+ --> src/pycell.rs
+  |
+  | pub struct PyRefMut<'p, T: PyClass<Frozen = False>> {
+  |                                    ^^^^^^^^^^^^^^ required by this bound in `PyRefMut`
+  = note: this error originates in the attribute macro `pymethods` (in Nightly builds, run with -Z macro-backtrace for more info)
+
```

Comment:
Given the corresponding test did not change i'm not sure where this one comes from and if it is expected

Reply:
I think what's happened is that the spans for the code in question are now mixed, so the error has duplicated onto both spans.

It's not really expected but I think it'll also be quite hard to fix without a bit of trial-and-error on spans. I think for now I'm inclined to leave this as-is and hope when we remove these warnings in a release or two it cleans up again. It is a touch ugly though. :(

Reply:
I see, sounds reasonable